client/src/pages/admin/pages/add-data-source.tsx

```diff
@@ -50,49 +50,47 @@ export const DataSourceForm = () => {
       };
 
       return (
-        <div>
-        <h1>Add data source</h1>
         <form ref={formRef} onSubmit={handleSubmit}>
         <input
             type="text"
             name="name"
             placeholder="Data Source Name"
-            className="input input-bordered input-success w-full max-w-xs"
+            className="input input-bordered input-success w-full max-w-xs text-white"
```

Comment:
I would change the input colour to be the opposite of the input background, so if your background colour is bg-base-100 it should be bg-base-content: https://daisyui.com/docs/colors/

What does the .input class set as default bg colour?